src/angular/popup-menu/popup-menu-list.component.ts

```diff
@@ -45,6 +45,7 @@ export class PopupMenuListComponent implements AfterContentInit {
         }
     }
     @Input() public className: any;
+    @Input() public relative: boolean;
```

Comment:
Since `relative` is only used as a CSS, it is possible to omit this Input, and instead use it as part of `className`.
By default, put the CSS directives of `.static` as defaults in the CSS of the popup-menu, and use only `.relative` class to override the necessary directives.